sms/send-sms.php

```diff
@@ -1,24 +1,22 @@
 <?php
+
+use Zend\Diactoros\Response\Serializer;
```

Comment:
Lines 3 and 4 aren't used in this example. Is it worth removing them?
Or if they are needed, would it be better renaming the `Request\Serializer` to something more descriptive than `asdf`?

Reply:
Removed, was some testing code that accidentally got committed.